static/app/views/projectInstall/createProject.tsx

```diff
@@ -216,7 +221,7 @@ function CreateProject() {
 
   const canSubmitForm =
     !inFlight &&
-    team &&
+    (team || accessTeams.length === 0) &&
```

Comment:
Does `(team || accessTeams.length === 0)` always default to True b/c I see team is set based on `accessTeams`

Reply:
Team comes from dropdown input. It's set to the first one  in the beginning but I think as they change the dropdown they can set it to empty again.

Reply:
So basically this is true if they don't have any team to select, and if they have teams to select they selected at least one.
It will be false if someone has access to multiple teams but didn't select any of them.

Reply:
ah ok, thanks for clarifying